apps/editing-toolkit/editing-toolkit-plugin/editor-gutenboarding-launch/index.ts

```diff
@@ -82,7 +94,7 @@ function updateEditor() {
 			 * the control launch flow gets replaced by the "Step by Step" launch flow,
 			 * displayed in a modal on top of the editor (no redirect needed)
 			 */
-			const shouldOpenStepByStepLaunch = isGutenboarding;
+			const shouldOpenStepByStepLaunch = isGutenboarding && getIsFreePlan();
```

Comment:
Technically, `getIsFreePlan()` can be called while the interval is still running, and therefore it could return false positives. 

Altough I don't think this issue should be a blocker, since it's a very rare edge case (the selector would need to be very slow, and the user would need to click on "Launch" very quickly).
Plus, even in that case, the worst thing that would happen is that the user is taken to the "control" launch flow, instead of being shown the step by step launch flow.

Reply:
Yeah, probably disabling the _Launch_ button while we get site data would have been an ideal solution but keep in mind this is just temporary (because the Step-by-step launch can't handle the existing paid plan).

Reply:
Got it. Thank you for the explanation!